app/src/main/java/com/example/dispatcher/data/api/ApiConfigManager.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.dispatcher.data.api
+
+import com.ibm.icu.util.ULocale
+
+object ApiConfigManager {
+    const val BASE_URL = "https://newsapi.org/v2/"
+    const val TOP_HEADLINES_ENDPOINT = "top-headlines"
+    private var countryCode = "us"
+
+    fun getCountryCode(): String {
+        return countryCode
+    }
+
+    fun updateCountryCode(newCountryCode: String) {
```

Comment:
i saw in the documentation, and you are correct - only US is supported, so there's no need in updating the country code. you can delete relevant funcs and logic of the country code.